packages/react-dom/src/shared/DOMProperty.js

```diff
@@ -228,8 +251,8 @@ var HTMLDOMPropertyConfig = {
     loop: HAS_BOOLEAN_VALUE,
     // Caution; `option.selected` is not updated if `select.multiple` is
     // disabled with `removeAttribute`.
-    multiple: MUST_USE_PROPERTY | HAS_BOOLEAN_VALUE,
-    muted: MUST_USE_PROPERTY | HAS_BOOLEAN_VALUE,
+    multiple: HAS_BOOLEAN_VALUE,
+    muted: HAS_BOOLEAN_VALUE,
```

Comment:
I think this is probably fine, but why the change?

Reply:
Not sure what you mean? I removed the flag. https://github.com/facebook/react/pull/11733/commits/c9ca797dfa3e3522fd3fcb432e3ffb70b15d0215

Easier to read as individual commits.

Reply:
Got it. Thank you!

Reply:
@gaearon at this point it looks like there aren't any properties that contain more than one flag. We could probably stop using bitmasks and just use an enum, remove `checkMask` and use a single property in `propertyInfo` instead of 6.

Reply:
My plan is to get rid of `propertyInfo` completely (maybe). This PR is just very iterative.

Reply:
This is a good observation though.